src/nativescript/client.ts

```diff
@@ -49,4 +49,8 @@ export class Client extends CoreClient {
     client.activeUserStorage = new ActiveUserStorage();
     return client;
   }
+
+  static sharedInstance() {
```

Comment:
Why was this added if it just calls `super.sharedInstance()`?

Reply:
I must have committed this by accident - I was trying out some stuff, doesn't even look like valid syntax. I'll remove it as part of the refactoring ticket. Good catch.